ninio-snmp/src/main/java/com/davfx/ninio/snmp/encryption/PrivacyProtocol.java

```diff
@@ -0,0 +1,15 @@
+package com.davfx.ninio.snmp.encryption;
+
+public interface PrivacyProtocol {
+
+    String name();
+
+    String protocolId();
+
+    String protocolClass();
```

Comment:
`id()` and `class()` should be enough

Reply:
Yeah.. makes sense.
Maybe another method name could be appropriate.
How about `family()` or `category()` ?